queries.sql

```diff
@@ -8,3 +8,41 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = true;
 SELECT * FROM animals WHERE name != 'Gabumon';
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+START TRANSACTION;
+UPDATE animals
+SET species = 'unspecified';
+SELECT species FROM animals;
+ROLLBACK;
+SELECT species FROM animals;
+
+START TRANSACTION;
+UPDATE animals
+SET species = 'digimon' WHERE name LIKE '%mon';
+SELECT species FROM animals;
+UPDATE animals
+SET species = 'pokemon' WHERE name NOT LIKE '%mon';
+SELECT species FROM animals;
+COMMIT;
+SELECT species FROM animals;
+
+START TRANSACTION;
+DELETE FROM animals;
+ROLLBACK;
+
+START TRANSACTION;
+DELETE FROM animals WHERE date_of_birth > '2022-01-01';
+
+SAVEPOINT update_weights;
+UPDATE animals SET weight_kg = weight_kg * -1;
+ROLLBACK TO update_weights;
+UPDATE animals SET weight_kg = weight_kg * -1 WHERE weight_kg < 0;
+COMMIT;
+
+SELECT COUNT(*) AS total_animals FROM animals;
+SELECT COUNT(*) AS non_escaping_animals FROM animals WHERE escape_attempts = 0;
+SELECT AVG(weight_kg) AS average_weight FROM animals;
+SELECT neutered, SUM(escape_attempts) AS total_escape_attempts FROM animals GROUP BY neutered;
```

Comment:
- [ OPTIONAL ] Great job so far but I think it would be great if you just put max instead of sum you get the animals that's right but I think you should also add max instead of sum to get the most animal escaped 😊🔥 you get the first two animals but you should get one so kindly update it